src/Corax/IndexFieldsMapping.cs

```diff
@@ -30,11 +30,37 @@ public IndexFieldsMapping(ByteStringContext context)
         _fieldsList = new List<IndexFieldBinding>();
     }
 
+    
+    private const int I64Postfix = 875972909; //"-I64"
+    private const int DblPostfix = 1818379309; //"-Dbl"
```

Comment:
This is a bit different from what we do for Lucene. Where the suffixes are `_L` and `_D`. I don't think it matters, but wanted to point it out.

Reply:
Changed that to match Lucene.